src/libs/address.js

```diff
@@ -1,30 +1,41 @@
 import IdunnPoi from '../adapters/poi/idunn_poi';
 
 /**
- * Format an address given an address object (name, city, country, label, and other regions)
+ * Filter an address and return an array with the relevant items
  * @param {*} address - an address object
  */
-export function format(address) {
-  if (!address) {return '';}
+export function filter(address) {
+
+  if (!address) {return [];}
+
+  const city = address.postcode && address.city
```

Comment:
Will this be correct accross countries ?

Reply:
Probably not. But this is probably good enough for now, as we are not sure yet about how this formatted address should be used in the application. In the longer term the API could be responsible for providing the appropriate format depending on the country.

Reply:
And I would name this `cityAndPostcode` to be more explicit